### PR TITLE
Document version property

### PR DESCRIPTION
**Pull request by davidtheclark**
Ref https://github.com/mapbox/mapbox-gl-js/issues/2851#issuecomment-232123646

### CODE REVIEW

File: js/ui/map.js [hunk 1]
```diff
@@ -1225,18 +1225,24 @@ function removeNode(node) {
...
-  *   be dramatically worse than expected (i.e. a software renderer would be used).
-  * @return {boolean}
-  * @example
-  * mapboxgl.supported() // = true
-  */
+/**
+ * Gets the version number of Mapbox GL JS.
+ *
+ * @var version
+ */
```
Comment (lucaswoj) [inline js/ui/map.js]: Can we declare that `version` is a `string`?

Comment (lucaswoj) [inline js/ui/map.js]: Can we provide any additional context here for users who don't immediately make the leap from `version number` to git tags, npm, GitHub releases, etc

Comment (davidtheclark) [inline js/ui/map.js]: I do not know of a way to declare that a `var` is a string. If I add a type, e.g. `@var version {string}`, compilation for that item seems to fail. Do you know of another way this should be marked with JSDoc?

Regarding context, what do you think of this? ` To learn more about the differences between published versions of Mapbox GL JS, have a look at [the changelog](https://github.com/mapbox/mapbox-gl-js/blob/master/CHANGELOG.md).` Sufficient? I guess I'm not sure what else to say here ... open to ideas.

**Comment by lucaswoj** [comment]
Looks like `@var` is intended to support a type parameter: http://usejsdoc.org/tags-member.html. It is good to capture the metadata even if this doesn't show up in our documentation theme as-written. 

I've pushed a few commits. 🚢 if this looks good to you!

**Comment by davidtheclark** [comment]
Thanks, @lucaswoj, and sorry for my obtuseness --- I just put the type in the wrong place (；一_一). What you've done makes sense.

**Pull request merged**